### DISCUSSION TITLE
Is there a way to do pos tagging on tokenized sentence?

### DISCUSSION THREAD

**Discussion by SilasTHU** [explosion/spaCy#10442] [Help: Coding & Implementations] [answered]
I know how to do pos tagging in the pipeline way.

I have a list, of which each element is a token of the sentence. Is there a way to do pos tagging on that list?

Thank you.

**Comment by polm** [comment, marked as answer]
You can create a Doc directly from a list of words to skip the tokenizer, and then pass that to your pipeline. See [the docs](https://spacy.io/api/doc#init).

**Reply by SilasTHU** [reply]
It worked. Thank you!
